doc/source/reference/ufuncs.rst

```diff
@@ -109,7 +109,7 @@ The output of the ufunc (and its methods) is not necessarily an
 :class:`ndarray`, if all input arguments are not :class:`ndarrays <ndarray>`.
 Indeed, if any input defines an :obj:`~class.__array_ufunc__` method,
 control will be passed completely to that function, i.e., the ufunc is
-`overridden <ufuncs.overrides>`_.
+`overridden <https://www.numpy.org/neps/nep-0013-ufunc-overrides.html>`_.
```

Comment:
I think this was intended to be a link to the ["Overriding Ufunc Behavior"](https://numpy.org/devdocs/reference/ufuncs.html#overriding-ufunc-behavior) section of the same document, rather than an external reference to the NEP.

Reply:
Agreed. Fixing original link.